tests/test_views.py

```diff
@@ -67,6 +69,29 @@ def test_success(self):
         self.assertIn('access', res.data)
         self.assertIn('refresh', res.data)
 
+    def test_update_last_login(self):
+        self.view_post(data={
+            User.USERNAME_FIELD: self.username,
+            'password': self.password,
+        })
+
+        # verify last_login is not updated
+        user = User.objects.get(username=self.username)
+        self.assertEqual(user.last_login, None)
+
+        # verify last_login is updated
+        with override_api_settings(UPDATE_LAST_LOGIN=True):
+            reload(serializers)
+            self.view_post(data={
+                User.USERNAME_FIELD: self.username,
+                'password': self.password,
+            })
+            user = User.objects.get(username=self.username)
+            self.assertIsNotNone(user.last_login)
+            self.assertGreaterEqual(datetime.now(), user.last_login)
```

Comment:
```suggestion
            self.assertGreaterEqual(timezone.now(), user.last_login)
```

Use `django.utils.timezone` for datetime (avoid TZ naive datetime objs)